src/Select/index.js

```diff
@@ -62,10 +63,14 @@ const Select = React.forwardRef(
             {...rest}
           >
             <option value="" disabled>
-              Choose one
+              {placeholder || 'Choose one'}
```

Comment:
Ideally I think we should use `defaultProps` here

Reply:
good idea!